tests/draft-next/dynamicRef.json

```diff
@@ -205,47 +205,75 @@
     {
         "description": "multiple dynamic paths to the $dynamicRef keyword",
         "schema": {
-            "$schema": "https://json-schema.org/draft/next/schema",
+            "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
Using draft 2020-12 here causes my validator to skip `propertyDependencies`.  It should be `draft/next`

Reply:
Thank you. I did test it, but that doesn't help if I don't commit the version I tested :man_facepalming:.

Reply:
Fixed.